elasticsearch-details/groups-rule-mapping.md

```diff
@@ -0,0 +1,76 @@
+# External to local groups mapping 
+
+`Groups` rule matches request which user is assigned to at least one of its groups. Groups are being assigned to a username (or username pattern) in `users` section of ROR's configuration. Each element of the section requires also an authentication method (like eg. `auth_key` or `ldap_authentication`). But we can also use an authorization rule in this section beside the mentioned authentication rule (or one rule which provides both - the authentication and authorization altogether). Thanks to this, we gain a possibility to map external groups to local ones. Let's see it on an example.
+
+## Example
+
+```yaml
+readonlyrest:
+
+  access_control_rules:
+  - name: "Viewer block"
+    indices: ["logstash-viewers*"]
+    groups: ["viewers"]
+
+  - name: "DevOps block"
+    indices: ["logstash-devops*"]
+    groups: ["devops"]
+
+  [...]
+
+  users:
+  - username: "joe"
+    groups: ["editors"]
+    auth_key: joe:password
+
```

Comment:
\# Externally authenticated user + authorization via group mapping
\# Users belonging to "external_group1" OR "external_group2" are authorized as "viewers" and "editors" in the ACL.